.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 50
+    reviewers:
+      - "kr-project/model-and-backend-epic-area"
```

Comment:
Actually @chrisleck should we move this to @kr-project/ecosystem? This fork is some utility only used by the API